docs/user-guide.md

```diff
@@ -109,18 +110,45 @@ Initial steps similar to those of the Single Instance installation.
 
    `./install-oracle.sh --help`
 
-1. Run installation. Only IP address of the first cluster node is required:
+1. Create cluster config file. The file `cluster_config.json` is provided as template
+   and should be updated accordingly.
+
+1. Run installation pointing to the file for planned cluster configuration:
 
    ```bash
    ./install-oracle.sh \
    --ora-swlib-bucket gs://[cloud-storage-bucket-name] \
    --backup-dest "+RECO" \
    --ora-swlib-path /u02/swlib/ \
    --ora-swlib-type gcs \
+   --cluster-type RAC \
+   --cluster-config cluster_config.json
+   ```
+
+## Command quick reference for DR deployments
+
+Before creation of standby database the primary host and its database should exist.
+
+The primary database creation can be performed using [Single Instance Deployments section](#command-quick-reference-for-single-instance-deployments)<br>
+without or set to NONE `--cluster-type` option.
```

Comment:
When you create the primary database, omit the `--cluster-type` option or set it to `NONE`. To create the primary database, see [Single Instance Deployments](#command-quick-reference-for-single-instance-deployments).